### PR TITLE
Single file Ftrack components

### PR DESCRIPTION
**Pull request by tokejepsen**
This allows for having single file outputs with the datamember instance.data["output"].

### CODE REVIEW

File: pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def process(self, context):
                         instance.data["families"]
                     )
                 )[0]
-                instance.data["outputPath"] = os.path.join(
+                instance.data["output"] = os.path.join(
```
Comment (iwootten) [inline pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py]: It's unclear from the variable name here what it represents.

**Review by iwootten — CHANGES_REQUESTED** [review]

File: pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py (hunk 1 above)
Comment (tokejepsen) [reply pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py]: Have you got a suggestion for something better?

Comment (iwootten) [reply pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py]: To me, the original name was more indicative of what it represented. Output could be any variable type - path would suggest a string. I'd lose the camelcasing though if anything.

Comment (tokejepsen) [reply pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py]: Cool, I can go back to the original variable name.

```mixedCase``` has been suggested by Marcus for any data members, which I have been sticking to.

Comment (iwootten) [reply pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py]: Really? Why is that? That's counter to PEP8 recommendations isn't it?

Comment (iwootten) [reply pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py]: In fact, elsewhere in this PR you use drop cased key names, I think we should stick to a single convention and in my mind it should be the same as the naming used for variables.

**Comment by tokejepsen** [comment]
What do you think now?

**Comment by iwootten** [comment]
Hey, this looks fine to me.

**Review by iwootten — APPROVED** [review]

**Pull request merged**